src/clarity-angular/layout/nav/_responsive-nav.clarity.scss

```diff
@@ -275,13 +275,6 @@ $clr-transition-style: ease;
 
                 .header-hamburger-trigger + .branding {
                     padding-left: 0;
-
-                    //deprecate .clr-icon, .logo
-                    .clr-icon,
-                    .logo,
-                    clr-icon {
```

Comment:
I think we need to keep it for now. Its going to be a breaking change for anyone using that markup structure. We had a slack discussion about not removing those things until we do the css standardization work.